src/Helper.ts

```diff
@@ -308,7 +317,18 @@ export class Helper {
       manifestGroup.addItem(<IMetadataItem>metadataItem);
     }
 
+    const requiredStatement: LabelValuePair | null = this.manifest.getRequiredStatement();

```

Comment:
There's a method in this class already called `getMostSpecificRequiredStatement`. Might you want to use that instead of the direct access to the manifest's statement? (Maybe there's a reason not to -- I'm really not sure -- just asking because I noticed that it exists).

Reply:
Thanks Demian. getMostSpecificRequiredStatement returns an ILabelValuePair rather than a LabelValuePair, which is what's needed to construct the MetadataGroup, so it would take some adaptation to make it fit here, and might not be necessary anyway: getMosSpecificRequiredStatement seems useful in cases where a range or canvas within a manifest also have a requiredStatement; as this helper is applicable to these different resources anyway, getRequiredStatement should do the same thing. Any other metadata for the same instance is drawn with this.manifest.getMetadata(), so at least this.manifest.getRequiredStatement() ensures that this function is all working from the same resource level. However I'm finding it difficult to disentangle so it probably deserves a closer look using manifests that have metadata/requiredStatements at the range or canvas level.

Reply:
Yeah, I'm not really too familiar with the context for these different levels of required statements.

I'm also not sure exactly what should be done about ILabelValuePair vs. LabelValuePair. My understanding is that everything prefixed with "I" is an interface, so that means that LabelValuePair should implement the interface ILabelValuePair, and I would expect that methods requiring that functionality should be typed on the interface instead of the specific class. But it's possible that pattern was abandoned at some point, or that I'm misremembering something.

In any case, I'll see if I can find some time to test this more directly either later today or early next week!